src/pages/in/embed/_id/button.vue

```diff
@@ -247,6 +247,11 @@ export default {
       this.isShowAltMode = true;
       window.addEventListener('message', this.onReceiveMessage, false);
     }
+
+    const { exp } = this.$route.query.exp;
+    if (exp && window.ga) {
+      window.ga('set', 'exp', exp);
```

Comment:
since we are using GTM instead of just ga, I think we need to set cookie too
i.e. https://github.com/likecoin/likecoin-button/blob/master/src/plugins/optimize.js

Reply:
is this mean we can just do once experiment at a time?

Reply:
this mean this qs will always override any current exp I guess